lib/src/judo_floating_card.dart

```diff
@@ -0,0 +1,210 @@
+              ),
+              _getText(messageStack.first.messageLevel),
+              Positioned(
+                left: 132,
+                bottom: 24,
+                child: ConstrainedBox(
+                  constraints: BoxConstraints(
+                    maxWidth: 280,
+                  ),
+                  child: Container(
+                    height: 24,
+                    child: Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 20),
+                      child: Text(
+                        messageStack.first.actionCallName ?? '',
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                          color: _getTextColor(messageStack.first.messageLevel, theme),
+                        ),
+                        overflow: TextOverflow.fade,
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+              Positioned(
+                  top: 2,
+                  right: 2,
+                  child: Material(
+                    type: MaterialType.transparency, // InkSplash occurs on the closest ancestor Material widget.
+                    child: IconButton(
+                      onPressed: closeAction,
+                      icon: Icon(
+                        Icons.close,
+                        size: 18,
+                      ),
+                      splashRadius: 20,
+                    ),
+                  )
+              )
+            ],
+          ),
+        ),
+      );
+    }
+    return SizedBox.shrink();
+
+  }
+
+  List<Widget> _floatingCardsStack(final List<FloatingMessage> messageStack) {
+    if (messageStack.length < 2) return [SizedBox.shrink()];
+    List<Widget> floatingCards = [];
+
+    for (int i = 1; i < messageStack.length; i++) {
+      floatingCards.insert(0,
+          Positioned(
+            top: 7 * i.toDouble(),
+            left: 7 * i.toDouble(),
+            child: Container(
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.vertical(
+                    bottom: Radius.circular(20)
+                ),
+                color: Color.fromRGBO(
+                    _getBackgroundColor(messageStack[i].messageLevel).red - 10 - i * 2,
+                    _getBackgroundColor(messageStack[i].messageLevel).green - 10 - i * 2,
+                    _getBackgroundColor(messageStack[i].messageLevel).blue - 10 - i * 2,
+                    0.7
+                ),
+              ),
+              width: width,
+              height: height,
+            ),
+          )
+      );
+    }
+
+    return floatingCards;
+  }
+
+  Widget _getText(MessageLevel messageLevel) {
+    double leftPosition = 132;
+
+    if (messageLevel == MessageLevel.info) {
+      leftPosition = 44;
+    }
+
+    return Positioned(
+        top: 24,
+        left: leftPosition,
+        child: Container(
+          height: 72,
```

Comment:
I'm worried about these hard-coded values. If we checked the widgets in different screen sizes and all was good, then it's okay. Otherwise these might need to be extracted as parameters as well.

Reply:
AppTheme used to be used this type of settings. 72 is the linelength but other places it is used as parameter!